src/Network/PacketHandlers.cs

```diff
@@ -6359,8 +6358,9 @@ string[] lines
                 }
                 else if (string.Equals(entry, "checkertrans", StringComparison.InvariantCultureIgnoreCase))
                 {
-                    applyCheckerTrans = true;
                     gump.Add(new CheckerTrans(gparams), page);
+                    var checkerTrans = gump.Children.Last();
```

Comment:
Declaring a `new Checkertrans()` out of the `gump.Add` method  and  use it instead of call  `.Last()` will look a way better

Reply:
good shout, fixed